chain_bridge.go

```diff
@@ -376,9 +377,17 @@ func (l *LndRouterClient) DeleteLocalAlias(ctx context.Context, alias,
 	return l.lnd.Router.XDeleteLocalChanAlias(ctx, alias, baseScid)
 }
 
+func (l *LndRouterClient) SubscribeHtlcEvents(
```

Comment:
nit: requires Godoc comment (for the same method in the `HtlcSubscriber` interface as well).